MAUI/Toolbar/overlay-toolbar.md

```diff
@@ -140,4 +140,184 @@ namespace ToolbarSample
 
 {% endtabs %}
 
-![overlay-toolbar](images/overlay-toolbar.gif)
\ No newline at end of file
+![overlay-toolbar](images/overlay-toolbar.gif)
+
+## Back icon customization
+The toolbar control support customization of back icon using properties BackIconColor[https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Toolbar.SfOverlayToolbar.html#Syncfusion_Maui_Toolbar_SfOverlayToolbar_BackIconColor], BackIconAlignment[https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Toolbar.SfOverlayToolbar.html#Syncfusion_Maui_Toolbar_SfOverlayToolbar_BackIconAlignment], BackIconTemplate[https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Toolbar.SfOverlayToolbar.html#Syncfusion_Maui_Toolbar_SfOverlayToolbar_BackIconTemplate], BackIconToolTipText[https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Toolbar.SfOverlayToolbar.html#Syncfusion_Maui_Toolbar_SfOverlayToolbar_BackIconToolTipText].
```

Comment:
Backiconaligment
Backiconcolor
BackIconToolTipText
BackIconTemplate

add in this order